docker/compose/docker-compose.yml

```diff
@@ -153,17 +249,31 @@ services:
     networks:
       - default
   syslog:
+    image: mozdef/mozdef_syslog
+    build:
+      context: ../../
+      dockerfile: docker/compose/mozdef_syslog/Dockerfile
+      cache_from:
+        - mozdef/mozdef_syslog
+        - mozdef_syslog:latest
     restart: always
-    command: bash -c 'sleep 95 && /usr/sbin/syslog-ng --no-caps -F'
+    command: bash -c '/usr/sbin/syslog-ng --no-caps -F'
```

Comment:
This container is dependent on the rabbitmq container/service being up, which has in the past, caused a timing related problem. I don't know if the curl command will work on port 5672 (amqp), or if we need to readd the sleep/write a script to monitor the port?

Reply:
yeah that probably doesnt work with curl

Reply:
i changed this to a bash command so it works with all and still no deps